build.sh

```diff
@@ -6,8 +6,5 @@ set -e
 set -x
 
 if [ "$1" == "release" ]; then
-  # TODO load keys from internal storage
-  # mvn clean install
-else
-  # mvn clean install
+  mvn clean install
```

Comment:
Previously the intention was to do `mvn clean install` regardless of `$1` being `"release"`. Now building only when `"release"`?